pkg/preparer/orchestrate.go

```diff
@@ -401,6 +401,12 @@ func (p *Preparer) installAndLaunchPod(pair ManifestPair, pod Pod, logger loggin
 		return false
 	}
 
+	err = pod.CreateCgroupForPod()
+	if err != nil {
+		logger.WithError(err).
+			Errorln("Pod Cgroup creation failed, proceeding")
```

Comment:
So if we fail to create cgroups we allow a pod to run on a machine with unrestricted resources? That seems like lying about the available resources on the machine and allowing it to be a bad neighbor.

What do you think about failing the install/launch and letting it retry (since the key would still be in intent)?

Reply:
This is a fair point, I didn't consider that the preparer would retry.